tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -2399,6 +2400,39 @@ def test_import(self, file_format: str) -> None:
         for i, tag in enumerate(tags):
             assert tag["value"] == new_tags[i]["value"]
 
+    @ddt.data(
+        "csv",
+        "json",
+    )
+    def test_import_plan(self, file_format: str) -> None:
+        """
+        Tests planning import a valid taxonomy file.
+        """
+        url = TAXONOMY_TAGS_IMPORT_PLAN_URL.format(pk=self.taxonomy.id)
+        new_tags = [
+            {"id": "tag_1", "value": "Tag 1"},
+            {"id": "tag_2", "value": "Tag 2"},
+            {"id": "tag_3", "value": "Tag 3"},
+            {"id": "tag_4", "value": "Tag 4"},
+        ]
+        file = self._get_file(new_tags, file_format)
+
+        self.client.force_authenticate(user=self.staff)
+        response = self.client.put(
+            url,
+            {"file": file},
+            format="multipart"
+        )
+        assert response.status_code == status.HTTP_200_OK
+
+        # Check if the taxonomy was not changed
+        url = TAXONOMY_TAGS_URL.format(pk=self.taxonomy.id)
+        response = self.client.get(url)
+        tags = response.data["results"]
+        assert len(tags) == len(self.old_tags)
+        for i, tag in enumerate(tags):
+            assert tag["value"] == self.old_tags[i].value
```

Comment:
Could you add this on a separate utility function? It is repeated in different tests.

Reply:
Thank you @ChrisChV! The code grew, and I overlooked that.
Done fc3a6daf0f2c9f3a9f7796d67fcc374212a11b03!